.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       # therefore a new base image chose instead to guarantee to
       # have a newer version >= 1.8.10 to avoid warnings
       # that related to the default behaviors or non-exist config options
-      - image: cimg/base:2021.05
+      - image: cimg/python:3.9
```

Comment:
Note that python 3.10 and 3.11 are also available. I chose 3.9 as most likely to be compatible with our other documentation dependencies.